MAUI/Chips/Customization.md

```diff
@@ -1806,4 +1806,6 @@ using Syncfusion.Maui.Core;
 
 N> The default value of ChipImageSize is [`18d`].
 
+## IsSelected
 
+The `IsSelected` property is a public property available in the [`SfChip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfChip.html). This property indicates whether a chip has been selected in choice and filter types.
```

Comment:
The `IsSelected` property in the [`SfChip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfChip.html) is used to indicate whether a chip has been selected in choice and filter types.

Reply:
Addressed the comments.